src/client/app/actions/groups.js

```diff
@@ -296,9 +302,14 @@ function submitGroupEdits(group) {
 					dispatch2(changeDisplayMode(DISPLAY_MODE.VIEW));
 				});
 			})
-			.catch(error => {
+			.catch(() => {
 				dispatch(markGroupInEditingNotSubmitted());
-				console.error(error);
+				dispatch(showNotification({
+					message: 'Failed to edit group',
+					level: 'error',
+					position: 'tr',
+					autoDismiss: 3
+				}));
```

Comment:
It's fine to show a notification, but it should also log to the console.

The changes to this file also have nothing to do with the admin panel, and should not really be in this PR.